### PR TITLE
Biblically Accurate TextureSet labels

### PR DESCRIPTION
**Pull request by JamesFrancoe**
Changed some labels to be more accurate based on research

### CODE REVIEW

**Comment by JamesFrancoe** [comment]
ignore the EdFile.MD stuff, Idk how to use github and I thought that it was only gonna be on my fork

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs [hunk 1]
```diff
@@ -38,16 +38,16 @@ public NuTexGenHdr Deserialize(BinaryReader reader, uint nuTexHdrVersion)
                 {
                     path = reader.ReadSized32NullTerminatedString();
 
-                    uint nuTexGenHdrUnknown1 = reader.ReadUInt32BigEndian();
+                    uint nuttype = reader.ReadUInt32BigEndian();
```
Comment (AcK77) [inline src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:41]: What about 
```suggestion
                    uint nuTexGenHdrType = reader.ReadUInt32BigEndian();
```
?

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs [hunk 2]
```diff
@@ -38,16 +38,16 @@ public NuTexGenHdr Deserialize(BinaryReader reader, uint nuTexHdrVersion)
...
                 }
 
                 if (nuTexHdrVersion == 12)
                 {
-                    uint nuAlignedBuffer = reader.ReadUInt32();
+                    uint Level = reader.ReadUInt32(); // Accurev internal level (unused)
```
Comment (AcK77) [inline src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:46]:
```suggestion
                    uint level = reader.ReadUInt32(); // Accurev internal level (unused)
```
What’s « Accurev » ?

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs [hunk 3]
```diff
@@ -38,16 +38,16 @@ public NuTexGenHdr Deserialize(BinaryReader reader, uint nuTexHdrVersion)
...
 
                     path = reader.ReadSized8NullTerminatedString();
 
-                    byte nuResourceId = reader.ReadByte();
+                    byte nuttypeAsU8 = reader.ReadByte();
```
Comment (AcK77) [inline src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:50]:
```suggestion
                    byte nuTexGenHdrTypeAsU8 = reader.ReadByte();
```
Maybe?

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs [hunk 4]
```diff
@@ -59,7 +59,7 @@ public NuTexGenHdr Deserialize(BinaryReader reader, uint nuTexHdrVersion)
 
                         path = reader.ReadSized16NullTerminatedString();
 
-                        byte unknown1 = reader.ReadByte();
+                        byte nuttypeAsU8 = reader.ReadByte();
```
Comment (AcK77) [inline src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:62]:
```suggestion
                        byte nuTexGenHdrTypeAsU8 = reader.ReadByte();
```

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTextureSet.cs [hunk 5]
```diff
@@ -20,7 +20,7 @@ public NuTextureSet Deserialize(BinaryReader reader)
                 throw new InvalidDataException($"{reader.BaseStream.Position:x8}");
             }
 
-            uint nuTextureSetHeaderUnknown1 = reader.ReadUInt32BigEndian();
+            uint TXSTHDRCount = reader.ReadUInt32BigEndian(); // always 1
```
Comment (AcK77) [inline src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTextureSet.cs:23]:
```suggestion
            uint nuTextureSetHeaderCount = reader.ReadUInt32BigEndian(); // Always 1
```

**Review by AcK77 — CHANGES_REQUESTED** [review]

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs (hunk 1 above)
Comment (JamesFrancoe) [reply src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:41]: nuttype is the internal name for the attributes of the NuTexture and has nothing to do with NuTexGenHdr

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs (hunk 2 above)
Comment (JamesFrancoe) [reply src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:46]: their internal source control system

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs (hunk 3 above)
Comment (JamesFrancoe) [reply src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:50]: explained before

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs (hunk 4 above)
Comment (JamesFrancoe) [reply src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:62]: explained before

File: src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs (hunk 1 above)
Comment (AcK77) [reply src/TTGamesExplorerRebirthLib/Formats/NuCore/NuTexGenHdr.cs:41]: Then maybe 
```suggestion
                    uint nuTextureType = reader.ReadUInt32BigEndian();
```

That made the name more accurate following the C# convention too (same for other comments)

**Review by AcK77 — APPROVED** [review]

**Pull request merged**